hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/index/TestHoodieIndexConfigs.java

```diff
@@ -132,47 +115,6 @@ public void testCreateIndexWithException() {
     assertTrue(thrown2.getMessage().contains("Unable to instantiate class"));
   }
 
-  public static class DummyHoodieIndex<T extends HoodieRecordPayload<T>> extends SparkHoodieIndex<T> {
```

Comment:
Maybe rewrite this for new APIs?

Reply:
@yihua this is actually only used by `testCreateDummyIndex()` to test index instantiation. the code path is already covered by other UTs so i removed this and the test too due to redundancy.

Reply:
Makes sense.